sort_image_files.sh

```diff
@@ -175,32 +169,24 @@ while read -r a_file_name; do
   day="${date_for_date_change:6:2}"
   path_with_year_month_day="${just_path}/${year}/${month}/${day}"
   path_with_year_month="${just_path}/${year}/${month}"
+  renamed_file_and_path="${a_file_name/IMG/$date_for_filename_change}"
```

Comment:
So this works fine as long as the letters `IMG` don't appear in any of the directory names in `${a_file_name}`. It might be a little safer to have

```
renamed_file="${just_filename/IMG/$date_for_filename_change}"
```

and then below where you do

```
just_filename=$(basename "${renamed_file_and_path}")
```

you can instead have

```
just_filename=$renamed_file
```

Reply:
Changes so default case for No Day subdir, No filename change so less code.